app/models/ingest_job.rb

```diff
@@ -487,35 +490,46 @@ def generate_success_email_array
     message
   end
 
-  # format an error email message body
+  # format an error email message body for users
+  #
+  # * *params*
+  #  - +email_type+ (Symbol) => Type of error email
+  #                 :user => High-level error message intended for users, contains only messages and no stack traces
+  #                 :detailed => Debug-level error messages w/ stack traces intended for SCP team
   #
   # * *returns*
   #  - (String) => Contents of error messages for parse failure email
-  def generate_error_email_body
-    error_contents = self.read_parse_logfile(self.error_filepath)
-    warning_contents = self.read_parse_logfile(self.warning_filepath)
-    message_body = "<p>'#{self.study_file.upload_file_name}' has failed during parsing.</p>"
+  def generate_error_email_body(email_type: :user)
+    case email_type
+    when :user
+      error_contents = self.read_parse_logfile(self.user_error_filepath)
+      message_body = "<p>'#{self.study_file.upload_file_name}' has failed during parsing.</p>"
+    when :detailed
```

Comment:
We refer to this type of email / log a few different ways:
* detailed (here)
* admin (elsewhere in this PR)
* dev (in [Ingest Pipeline code](https://github.com/broadinstitute/scp-ingest-pipeline/pull/141))

Having one (and only one) word for this type of email / log in both `single_cell_portal_core` and `scp-ingest-pipeline` would help.  The other type of email / log is more consistently referred to as a "user" type, which is nice.

Reply:
I agree about the `:detailed/:dev` discrepancy, but I would push back about the `admin` name.  That is referring to email delivery, and our convention of `user` vs `admin` describes the audience, rather than the function.  I'd like to keep that consistent.  However, I'll make the changes in `ingest_job.rb` regarding the log file naming so that it matches with `scp-ingest-pipeline`.

Reply:
Helpful context -- I agree `admin` makes sense in that case.  Thanks for consolidating `:detailed/:dev`.